iceoryx2-bb/container/src/queue.rs

```diff
@@ -117,337 +118,332 @@ use iceoryx2_pal_concurrency_sync::iox_atomic::IoxAtomicBool;
 
 /// Queue with run-time fixed size capacity. In contrast to its counterpart the
 /// [`RelocatableQueue`] it is movable but is not shared memory compatible.
-pub type Queue<T> = details::MetaQueue<T, GenericOwningPointer>;
+pub type Queue<T> = MetaQueue<T, GenericOwningPointer>;
 /// **Non-movable** relocatable queue with runtime fixed size capacity.
-pub type RelocatableQueue<T> = details::MetaQueue<T, GenericRelocatablePointer>;
+pub type RelocatableQueue<T> = MetaQueue<T, GenericRelocatablePointer>;
 
 #[doc(hidden)]
-pub mod details {
```

Comment:
Sorry for the huge diff. I didn't change the file content-wise, just removed the `details` module and used the type aliases where appropriate.